docs/md/iofogctl_delete_agent.md

```diff
@@ -31,8 +29,10 @@ iofogctl delete agent NAME
 ### Options inherited from parent commands
 
 ```
+      --detached           Use/Show detached resources
       --http-verbose       Toggle for displaying verbose output of API client
   -n, --namespace string   Namespace to execute respective command within (default "default")
+      --soft               Don't delete ioFog Stack from remote host
```

Comment:
Instead of referring to Agent stack on host, can we use "uninstall Agent from host"?